.update-copyright.conf

```diff
@@ -0,0 +1,17 @@
+[project]
+name: lesson-template
+vcs: Git
+
+[files]
+authors: yes
+files: no
+
+[copyright]
+short: {project} comes with ABSOLUTELY NO WARRANTY and is licensed under the MIT and CC BY 4.0 licenses.
+long: This file is part of {project}.
+
+  Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
```

Comment:
On Sun, Mar 15, 2015 at 07:52:50PM -0700, W. Trevor King wrote:
> Sun, Mar 15, 2015 at 07:10:58PM -0700, Greg Wilson:
> > Doesn't this all duplicate what's in the LICENSE file?
> 
> Yup.  But I like the explicitness of per-file copyright blurbs
> (which is really what #198 is about).

To avoid distracting the author-listing discussion here, I've rebased
the branch to avoid configuring copyright blurbs in this PR.  Those
options are now set only in #198, which is only about whether or not
we want per-file copyright blurbs.